setup.cfg

```diff
@@ -0,0 +1,5 @@
+[yapf]
+based_on_style = pep8
+spaces_before_comment = 4
+split_before_logical_operator = true
+column_limit = 120
```

Comment:
I would prefer to stick to the suggested PEP8 standard of 79/80 characters. Note that the source files already conformed to this line length.

Personally I quite like the readability benefits of a short line length (as it encourages semantic indentation). If it is good enough for the standard library it should be good for everyone ;-).

I know this topic is super controversial...